docs/features/secrets.adoc

```diff
@@ -194,11 +194,11 @@ secrets:
     file: ./secrets/jcasc_vault
 ```
 
-==== Using environment variables
+==== Using environment variables (implies security risks)
 
 Environment variables can be directly read by JCasC when loading configurations.
 Secrets can be also injected using an environment variables.
-Note that such approach implies security risks,
+Note that such approach implies *security risks*, the administrator in charge of the configuration must trust the whole content that is loaded,
```

Comment:
I am not sure this should be burried in the `secrets` section?

Reply:
Thanks James, in the aaad8f6, I tried to emphasize this important aspects.